src/navigations/root-stack.js

```diff
@@ -170,6 +152,14 @@ const AuthenticatedNavigator = () => {
     );
   };
 
+  const withSafeAreaView = (Component) => {
+    return (props) => (
+      <SafeAreaView>
+        <Component {...props} />
+      </SafeAreaView>
+    );
+  };
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

This new function `withSafeAreaView` wraps a component with `SafeAreaView`. However, it's not used anywhere in the provided code. If it's not used elsewhere, it should be removed to avoid unnecessary code.
```diff
-   const withSafeAreaView = (Component) => {
-     return (props) => (
-       <SafeAreaView>
-         <Component {...props} />
-       </SafeAreaView>
-     );
-   };
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->